packages/twenty-server/src/modules/messaging/message-import-manager/drivers/microsoft/providers/microsoft-client.provider.ts

```diff
@@ -17,16 +17,8 @@ export class MicrosoftClientProvider {
       'refreshToken' | 'id'
     >,
   ): Promise<Client> {
-    try {
```

Comment:
this was never catching anything since the client only fails once calling the API, not at the client init